ibis/backends/dask/tests/test_udf.py

```diff
@@ -260,7 +257,7 @@ def f(s):
         return s.sub(s.mean()).div(s.std())
 
     expected = df.groupby('key').c.transform(f).compute()
-    tm.assert_series_equal(result, expected)
+    tm.assert_series_equal(result, expected, check_names=False)
```

Comment:
these don't preserve?

Reply:
Not when you're using the udf on the dask backend directly like this. 

I could cheat and say "if we're running the udf on a single column then set the output name to that column"

Reply:
kk (just maybe a comment about this as its slightly unusual to not match like this)